src/xml.tsx

```diff
@@ -388,7 +339,7 @@ export function parse(source: string, middleware?: Middleware): JsxAST | null {
       props,
       children: [],
       parent: currentElement,
-      Tag: tags[tag] || missingTag,
+      Tag: getTag(tag as keyof typeof tags),
```

Comment:
Why do we need this change? Shouldn't tags from `tags.tsx` and `tags.web.tsx` be exactly the same?

Reply:
changed that, and restored the old version.